pkg/config/torch_compatibility_matrix.json

```diff
@@ -1,4 +1,64 @@
 [
+  {
+    "Torch": "2.4.0",
+    "Torchvision": "0.19.0",
+    "Torchaudio": "2.4.0",
+    "FindLinks": "",
+    "ExtraIndexURL": "https://download.pytorch.org/whl/cu124",
+    "CUDA": "12.4",
```

Comment:
So CUDA minor versions are not compatible? And there's no 12.2 or 12.3?

Reply:
Going from this compatibility matrix here: https://pytorch.org/

Also this is somewhat cargo-culted from the rest of the compatibility matrix.

![Screenshot 2024-08-21 at 4 34 19 PM](https://github.com/user-attachments/assets/37551d5b-f9c7-48b2-90e0-1e87417ae693)